configs/raspberrypi_config

```diff
@@ -95,4 +95,34 @@ function platform_modify() {
     run_and_log_cmd "sudo unlink work/rootfs/etc/rc3.d/S01resize2fs_once"
   fi
   run_and_log_cmd "sudo rm -f work/rootfs/etc/init.d/resize2fs_once"
+
+  #
+  # Resize the data partition to fill the remaining space, using parted, with systemd
+  #
+  log_info "Adding systemd init script to run parted and resize the data partition on boot"
+  log_info "to fill all the available space on the storage media"
+  run_and_log_cmd "sudo mkdir -p work/rpi/etc/systemd/system/"
+  run_and_log_cmd "sudo mkdir -p work/rootfs/etc/systemd/system/data.mount.wants/"
+  cat <<-EOF > work/rpi/etc/systemd/system/mender-grow-data.service
+  [Unit]
+  Description=Mender service to grow data partition size
+  DefaultDependencies=no
+  Before=data.mount
+  Before=systemd-growfs@data.service
+
+  [Service]
+  Type=oneshot
+  User=root
+  Group=root
+  ExecStart=/sbin/parted --script /dev/mmcblk0 resizepart ${MENDER_DATA_PART_NUMBER} 100%
```

Comment:
I think `/dev/mmcblk0` should not be hardcoded. But it doesn't look like mender-convert uses `MENDER_STORAGE_DEVICE`, which surprised me a bit. We only have `MENDER_STORAGE_DEVICE_BASE`. I suggest you use the value from `MENDER_STORAGE_DEVICE_BASE`, minus the `p`, if there is one (`/dev/sda` would not have it, for example).

Reply:
If we do this in a separate config file and is only going to be used for our converted image, I don't think it is an issue to hardcode `/dev/mmcblk0`

Reply:
I think I agree with @lluiscampos here. This is very one-use-case specific

Reply:
Actually, my idea was that it could be reusable outside RPi as well, but I agree that this is not important to have at all, and this simplifies it considerably. In that case I'm fine with hardcoding it.

But in that case I would move this into the `configs/images/raspberrypi3_raspbian_config` instead, since this is specific to our image.

Reply:
@lluiscampos what is your take on the `configs/images/raspberrypi3_raspbian_config` specification file and path?

Reply:
I think it belongs (temporary) to `raspberrypi3` but not necessary to our image only, so I would move it to an independent file but not in `../images`. For example `configs/raspberrypi3_data_expansion_config`.

If it is faster, let's discuss it in person and take a decision.